Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -42,8 +45,11 @@ void SFilledGraphWidget::OnImGuiDraw() const
 		const float xMaxValue = FMath::Max(XAxisValues);
 		const float yMaxValue = FMath::Max(InData);
 
+		ImPlot::PushStyleVar(ImPlotStyleVar_PlotPadding, ImVec2(20, 20));
+		ImPlot::PushStyleVar(ImPlotStyleVar_LabelPadding, ImVec2(10, 10));
```

Comment:
Hard coding these numbers is fine for now but we should maybe start thinking about ways of doing this more automatically in the future. For example, if the last number on the X Axis is 1000000 instead of 100 it might still get clipped.

Reply:
So I haven't done anything about this yet but I tried to see if there's another way to add padding, such as have the padding be around the tick values but there doesn't seem to be an easy way to do that. We should think about this at some point though, should I add a task for this?

Reply:
Yeah it would be good to look into this, but it can be for when we're bugfixing/ perfecting later on